test/python/aer/test_qasm_simulator.py

```diff
@@ -243,10 +238,10 @@ def test_qobj_save_load(self):
         snapshots = result.data('save_command')['snapshots']['statevector']
         self.assertEqual(set(snapshots), {'0', '1', '10', '11'},
```

Comment:
not blocking but why label the snapshots in this wat

Reply:
i mean the '0', '1', '10', '11'. not the form. If they are labeling experiments why not 0 1 2 3 or 00 01 10 or 11

Reply:
it's labeling different points within the same experiment. these are the qobj instructions, I couldn't decipher the labeling:
```
                {"name": "save", "params": [0]},
                {"name": "snapshot", "label": "", "type": "", "params": [0]},
                {"name": "h", "qubits": [0]},
                {"name": "save", "params": [1]},
                {"name": "snapshot", "label": "", "type": "", "params": [10]},
                {"name": "load", "params": [0]},
                {"name": "snapshot", "label": "", "type": "", "params": [1]},
                {"name": "load", "params": [1]},
                {"name": "snapshot", "label": "", "type": "", "params": [11]},
```

Reply:
your overthinking. i dont see a problem i just asking why these labels. But we have over discussed a non issue :-)

Reply:
I git-blame @chriseclectic